packages/text-input/src/utils/getLGTextInputTestUtils/getLGTextInputTestUtils.ts

```diff
@@ -0,0 +1,124 @@
+import { getByLgId, queryByQuerySelector } from '@lg-tools/test-harnesses';
+
+import {
+  LGID_FORM_FIELD_ERROR_MESSAGE,
+  LGID_FORM_FIELD_OPTIONAL,
+} from '@leafygreen-ui/form-field';
+import { LGID_DESCRIPTION, LGID_LABEL } from '@leafygreen-ui/typography';
+
+import { LGID_TEXT_INPUT } from '../../constants';
+
+import { LGTextInputTestUtilsReturnType } from './getLGTextInputTestUtils.types';
+
+export const getLGTextInputTestUtils = (
+  lgId = LGID_TEXT_INPUT,
+): LGTextInputTestUtilsReturnType => {
+  /**
+   * Queries the DOM for the element using the `data-lgid` data attribute.
+   * Will throw if no element is found.
+   */
+  const element = getByLgId(lgId);
+
+  /**
+   * Queries the `element` for the label element. Will return `null` if the label is not found.
+   */
+  const label = queryByQuerySelector<HTMLElement>(
+    element,
+    `[data-lgid="${LGID_LABEL}}"]`,
```

Comment:
Not sure why this is causing tests to fail. It works inside `TextArea`.
Switching back to `[data-lgid="lg-label"]` works.

Reply:
it was the double quotes!